tests/test_io.py

```diff
@@ -55,13 +55,47 @@ def test_load_scan_spm(load_scan: LoadScan) -> None:
 #     assert test_filters_random.pixel_to_nm_scaling == expected
 
 
-def test_load_scan_get_data(load_scan: LoadScan) -> None:
+@pytest.mark.parametrize(
```

Comment:
Can include the parameters for the specific file types to ensure they are properties of a `LoadScan` object after calling `get_data()` method (this is somewhat duplicating the `test_load_scan_{spm,ibw}` tests though.

```
@pytest.mark.parametrize(
    "load_scan_object, filename, image_type, image_shape, image_sum, pixel_type, pixel_to_nm_scaling, suffix",
    [
        ("load_scan", "minicircle", np.ndarray, (1024, 1024), 30695369.188316286, float, 0.4940029296875, ".spm"),
        ("load_scan_ibw", "minicircle2", np.ndarray, (512, 512), -218091520.0, float, 1.5625, ".ibw"),
    ],
)
def test_load_scan_get_data(
    load_scan_object: LoadScan,
    filename: str,
    image_type,
    image_shape: tuple,
    image_sum: float,
    pixel_type,
    pixel_to_nm_scaling: float,
    suffix: str,
    request,
) -> None:
    """Test the LoadScan.get_data() method."""
    scan = request.getfixturevalue(load_scan_object)
    scan.get_data()
    assert isinstance(scan.filename, str)
    assert scan.filename == filename
    assert isinstance(scan.suffix, str)
    assert scan.suffix == suffix
    assert isinstance(scan.image, image_type)
    assert scan.image.shape == image_shape
    assert scan.image.sum() == image_sum
    assert isinstance(scan.pixel_to_nm_scaling, pixel_type)
    assert scan.pixel_to_nm_scaling, pixel_to_nm_scaling
```

Reply:
I was only trying to add to the test that was already in place. 

Would this be the preferred testing function for the `get_data` and `load_scan_spm.ibw` functions too?